user_modules/shared_testcases/commonPreconditions.lua

```diff
@@ -197,19 +221,28 @@ function Preconditions:Connecttest_without_ExitBySDLDisconnect(FileName)
 	os.execute(  'cp ./modules/connecttest.lua  ./user_modules/'  .. tostring(FileName))
 
 	-- remove quit(1) after SDL disconnect
-	f = assert(io.open('./user_modules/'  .. tostring(FileName), "r"))
+	local f = assert(io.open('./user_modules/'  .. tostring(FileName), "r"))
 
-	fileContent = f:read("*all")
+	local fileContent = f:read("*all")
 	f:close()
 
-	local patternDisconnect = "print%(\"Disconnected%!%!%!\"%).-quit%(1%)"
-	local DisconnectMessage = fileContent:match(patternDisconnect)
-	if DisconnectMessage == nil then
-		print(" \27[31m 'Disconnected!!!' message is not found in /user_modules/" .. tostring(FileName) .. " \27[0m ")
-	else
-		fileContent  =  string.gsub(fileContent, patternDisconnect, 'print("Disconnected!!!")')
+	local is_found = false
+	for i = 1, #pattern_exit_codes do
+		local patternDisconnect = "print%(\"Disconnected%!%!%!\"%).-quit%("..pattern_exit_codes[i].."%)"
+
+		local DisconnectMessage = fileContent:match(patternDisconnect)
+		
+		if( DisconnectMessage ~= nil )then 
+			fileContent  =  string.gsub(fileContent, patternDisconnect, 'print("Disconnected!!!")')
+			is_found = true
+			break
+		end 
 	end
 
+	if(is_found == false) then
+		print(" \27[31m 'Disconnected!!!' message is not found in /user_modules/" .. tostring(FileName) .. " \27[0m ")
+	end	
+
 	f = assert(io.open('./user_modules/' .. tostring(FileName), "w+"))
 	f:write(fileContent)
 	f:close()
```

Comment:
@dboltovskyi, I didn't put it in common function because I didn't want to make any sensitive change of commonPreconditions. The purpose of update is to resolve problem with execution on CI.